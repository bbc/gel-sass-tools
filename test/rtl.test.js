describe('rtl()', () => {
  it('uses ltr when the $rtl variable is not set', async () => {
    const css = await global.compileCSS('test/data/rtl/left.scss');

    expect(css).toMatchSnapshot();
  });

  it('switches to rtl when the $rtl variable is set to `true`', async () => {
    const css = await global.compileCSS('test/data/rtl/right.scss');

    expect(css).toMatchSnapshot();
  });
});
