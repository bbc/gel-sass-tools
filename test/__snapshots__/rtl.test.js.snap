// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rtl() switches to rtl when the $rtl variable is set to \`true\` 1`] = `
".rtl {
  margin-left: 10px;
  margin-right: 10px;
  border-left: 10px;
  border-right: 10px;
  border-left-width: 10px;
  border-right-width: 10px;
  border-right-color: red;
  border-left-color: blue;
  padding-left: 10px;
  padding-right: 10px;
  left: 10px;
  right: 10px;
}"
`;

exports[`rtl() uses ltr when the $rtl variable is not set 1`] = `
".ltr {
  margin-right: 10px;
  margin-left: 10px;
  border-right: 10px;
  border-left: 10px;
  border-right-width: 10px;
  border-left-width: 10px;
  border-left-color: red;
  border-right-color: blue;
  padding-right: 10px;
  padding-left: 10px;
  right: 10px;
  left: 10px;
}"
`;
